util/quality-measures-additional-info.json

```diff
@@ -0,0 +1,1217 @@
+[
+  {
```

Comment:
Ok - it's worth adding a note that there is manual intervention involved in the README.